build.ps1

```diff
@@ -34,16 +34,19 @@ function ExecSafe([scriptblock] $cmd) {
 }
 
 # If dotnet CLI is installed globally and it matches requested version, use for execution
-if ($null -ne (Get-Command "dotnet" -ErrorAction SilentlyContinue) -and `
+if ($null -eq $env:TEAMCITY_VERSION -and $null -ne (Get-Command "dotnet" -ErrorAction SilentlyContinue) -and `
      $(dotnet --version) -and $LASTEXITCODE -eq 0) {
     $env:DOTNET_EXE = (Get-Command "dotnet").Path
 }
 else {
     # Download install script
-    $DotNetInstallFile = "$TempDirectory\dotnet-install.ps1"
-    New-Item -ItemType Directory -Path $TempDirectory -Force | Out-Null
-    [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
-    (New-Object System.Net.WebClient).DownloadFile($DotNetInstallUrl, $DotNetInstallFile)
+    #$DotNetInstallFile = "$TempDirectory\dotnet-install.ps1"

```

Comment:
Should these commented lines be dropped?

Reply:
Oh did this file come from some other place?

Reply:
I kept these as commented out lines because they reference the initial default implementation of managing dotnet-install. Unfortunately, older versions of Windows don't enable TLS 1.2 in powershell by default, so the downloaded script fails (which is why we have a local copy that enables TLS 2.1)